app/app/pages/Welcome/Welcome.css

```diff
@@ -0,0 +1,74 @@
+.appCover {
+  position: relative;
+  min-height: 100vh;
+  display: flex;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  background-image: linear-gradient(to bottom, #9b2463, #312461);
+}
+.appBgPattern{
+  opacity: 0.1;
+  width:720px;
+  height: 456px;
+  z-index: 1;
+  position: absolute;
+  background: url("../../assets/images/app-bg.png") no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+.appContent{
+  max-width: 390px;
+  margin: 0 auto;
+  z-index: 2;
+  position: relative;
+
+}
+.appContent h1 {
+  color: #fff;
+  font-size:24px;
+  font-weight: 600;
+  text-align: center;
+  letter-spacing: 0.01em;
+  text-shadow: 0 0.5px 1px rgba(0, 0, 0, 0.5);
+}
+.appContent h4 {
+  color: #fff;
+  font-size:16px;
+  text-shadow: 0 0.5px 1px rgba(0, 0, 0, 0.5);
+  margin: 16px 0 28px;
+}
+.appContent a{
+  display: block;
+  font-weight: bold;
+  min-height: 40px;
+  max-width: 114px;
+  padding:8px 17px;
+  margin: 0 auto;
+  border-radius: 4px;
+  cursor: pointer;
+  text-transform: none;
+  color: #fff;
+  box-shadow: inset 0 0.3px 0.5px 0 #c25995;
+  background-image:linear-gradient(to bottom, #7c2463, #552462);
+}
+.logo{
+  height: 156px;
+  width:240px;
+  margin: 0 auto 0;
+  background: url("../../assets/images/Logo.png") no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+@media only screen and (min-device-pixel-ratio: 2),
+  only screen and (min-resolution: 192dpi),
+  only screen and (min-resolution: 2dppx) {
+  .appBgPattern {
+    background: url("../../assets/images/app-bg@2x.png") no-repeat;
```

Comment:
Since this asset is not used elsewhere, can we place it in the same directory (package by feature)?